packages/pap/pap.pacscript

```diff
@@ -0,0 +1,15 @@
+name="pap"
+version="0.13.2"
+url="https://github.com/talwat/pap/archive/v${version}.tar.gz"
+description="Swiss army knife for minecraft servers"
+maintainer="Talwat <talwat321@gmail.com>"
+build_depends="golang-go"
+incompatible=('debian:bullseye')
+
+build() {
+  make
```

Comment:
```suggestion
  make -j"$(nproc)"
```

Reply:
I'm not sure golang supports multithreaded compilation. I'm using a Makefile in case in the future I need to add any special build flags or whatever.

Reply:
It does, but it does it on its own with `go build` rather than through make.